docs/source/developer-guidelines/icons.md

```diff
@@ -8,7 +8,10 @@ html_meta:
 
 # Icons
 
-Volto has pre-defined set of SVG icons from Pastanaga UI icon system and are available under [Volto icons](https://github.com/plone/volto/tree/master/src/icons).
+Volto has a pre-defined set of SVG icons from the Pastanaga UI icon system.
+They are available under [Volto icons](https://github.com/plone/volto/tree/master/src/icons).
+
+The following example shows how to display one of these icons.```
```

Comment:
It looks like some trailing backticks snuck in here.

```suggestion
The following example shows how to display one of these icons.
```

Reply:
Done